Documentation/Configuration.md

```diff
@@ -58,6 +58,16 @@ top-level keys and values:
     argument, forcing the entire argument list to be laid out vertically.
     If false (the default), arguments will be laid out horizontally first, with
     line breaks only being fired when the line length would be exceeded.
+    
+*   `prioritizeKeepingFunctionOutputTogether` _(boolean)_: Determines if 
```

Comment:
Interesting historical note: this was the original wrapping behavior that we had implemented by default. That was also back when we forced every argument to be on its own line (`lineBreakBeforeEachArgument = true`), so we ended up removing this logic because we felt it was too aggressive in the case of a function that had a lot of arguments but only needed to wrap because of the output clause—in other words, the output clause would force the `)` down onto its own line, which would also force every argument onto its own line.

That's less of an issue with `lineBreakBeforeEachArgument = false`, so having a knob to re-enable this behavior doesn't seem too bad.